sys-devel/gettext/gettext-0.22.5.ebuild

```diff
@@ -113,6 +116,12 @@ src_prepare() {
 	if use elibc_musl || use elibc_Darwin; then
 		eapply "${FILESDIR}"/${PN}-0.21-musl-omit_setlocale_lock.patch
 	fi
+
+	if use elibc_musl ; then
+		QA_CONFIG_IMPL_DECL_SKIP+=( # bug #898570
+			fpurge
+		)
+	fi
```

Comment:
You are assuming that this should be skipped, but I don't know why. That check genuinely misfires in some scenarios, it is fixed in https://lists.gnu.org/archive/html/bug-gnulib/2024-11/msg00212.html

e.g. it is not cross-safe. I'm not sure it makes sense to flag it as a *false positive*.

It is fixed in gettext 0.23.

Reply:
I don't think this makes any sense as an argument. I don't care what you grep in the musl source code.

Upstream gnulib accepted this as a bug and agreed with my assessment that it will incorrectly diagnose the availability of the thing it is checking for, and as a direct consequence of this QA warning they adapted their code to fix a compilation error only applicable to certain scenarios.

Why are you disagreeing with me and with upstream gnulib / gettext (they are maintained by the same person)?

Why is it so important to you to inaccurately annotate an ***old*** version of this ebuild to silence a valid warning? If you are so concerned about fpurge then the correct answer is to backport the bug fix, not silence it?